src/js/dynload.ts

```diff
@@ -201,5 +201,7 @@ export async function loadDynlibsFromPackage(
   }
 }
 
-API.loadDynlib = loadDynlib;
-API.loadDynlibsFromPackage = loadDynlibsFromPackage;
+if (typeof globalThis.API !== "undefined") {
```

Comment:
You'd usually either want the condition to be `typeof API !== "undefined"` or `globalThis.API != undefined`. Or maybe just `if (globalThis.API)`. 

But why would `globalThis.API` be defined? Is this for testing? If so could you add a comment?

Reply:
`typeof API !== "undefined"` worked. Thanks!